.travis.yml

```diff
@@ -9,10 +9,10 @@ gemfile: crowbar_framework/Gemfile
 install:
   - cd crowbar_framework
   - bin/bundle install
-  - bin/rake db:create db:migrate
+  - SKIP_CHECKS=yes bin/rake db:create db:migrate
```

Comment:
I would prefer this as an environment variable for travis as we will never care about these checks on travis.